ovs/extensions/services/upstart.py

```diff
@@ -91,18 +92,27 @@ def add_service(name, client, params=None, target_name=None, additional_dependen
             client.file_write('/etc/init/{0}.conf'.format(target_name), template_file)
 
     @staticmethod
-    def get_service_status(name, client):
+    def get_service_status(name, client, return_output=False):
```

Comment:
@toolslive 
Type checking is not "standard" python code, so we don't use it.
Technically, the test "if return_output" has the same result as "if return_output is True" but the second is more restrictive and expressive (as we want a boolean); the first test will pass with any "True-ish" value.
As for the "type" of the return value, that's not important. Making the return_output by default False, means existing clients don't have to change their code when calling these methods, so the impact is minimal; only in places where we explicitly need the output, we pass True and expect a tuple (boolean, string) instead of a string (and also no type-checking or length-checking  is needed as the method is expected to return a string when return_output is False, return a tuple when return_output is True, or raise an exception when it can't return anything).
And if a client changes its mind about the output, they will have to explicitly ask it (pass True to return_output) and be prepared to receive a tuple.